scripts.js

```diff
@@ -106,5 +106,17 @@ const filterYear = () => {
     );
   });
 };
+const mapNames = () => {
```

Comment:
`inventorsFirstAndLastNames`, also the previous function should be called something like `bornInThe1500s` or similar